File: src/main/java/itmo/is/project/mapper/module/storage/StorageModuleMapper.java
package itmo.is.project.mapper.module.storage;

import itmo.is.project.dto.module.storage.StorageModuleDto;
import itmo.is.project.mapper.EntityMapper;
import itmo.is.project.model.module.storage.StorageModule;
import org.mapstruct.Mapper;

@Mapper(uses = {StorageModuleBlueprintMapper.class, StoredResourceMapper.class})
public interface StorageModuleMapper extends EntityMapper<StorageModuleDto, StorageModule> {
}

File: src/main/kotlin/itmo/isproject/storage/mapper/StorageModuleMapper.kt
package itmo.isproject.storage.mapper

import itmo.isproject.storage.dto.StorageModuleDto
import itmo.isproject.shared.common.mapper.EntityMapper
import itmo.isproject.storage.model.StorageModule
import org.mapstruct.Mapper

@Mapper(uses = [StorageModuleBlueprintMapper::class, StoredResourceMapper::class])
interface StorageModuleMapper : EntityMapper<StorageModuleDto, StorageModule>
